_posts/2022-12-05-solidity-core-team-updates.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+published: true
+title: 'Solidity Core Team Updates'
+date: '2022-12-05'
+author: Solidity Team
+category: Announcements
+---
+
+More than two years have passed since we [introduced Solidity core team members](https://blog.soliditylang.org/2020/09/18/meet-the-team/) on the blog and we realized it is high time for some updates: Meet new team members, find out who moved on to other adventures and learn about recent changes in the team structure!
+
+Before we dive in, a reminder that the Solidity programming language and compiler are **open-source community projects**. This post dives into the core team that leads the development. Nevertheless, we cannot stress enough how invaluable community contributions and interactions are for Solidity. We value feedback and input and want to thank our community for the continued support!
+
+## Thank you to our alumni 💐
```

Comment:
```suggestion
## Thank you to our alums 💐
```

Reply:
pretty sure alumni is the plural form of alumna

Reply:
TIL! But since we just collectively realized it's a group of men in this case we'll keep alumni in this special case. However, for the record, I now know what "alum" is and will use it in future! :D